bcol-api/src/bcol_api/resources/bcol_payment.py

```diff
@@ -42,6 +43,13 @@ class AccountPayment(Resource):
     def post():
         """Create a payment record in BCOL."""
         try:
+            if _jwt.validate_roles([Role.STAFF.value, Role.EDIT.value]):
```

Comment:
cursios , why _jwt ;not just jwt without underscore?

Reply:
We started using the _ for this decorators, as this instance won't be passed to any other class/function.